ze/tracer_ze_helpers.include.c

```diff
@@ -798,6 +1048,14 @@ static void _load_tracer(void) {
     else if (verbose)
       fprintf(stderr, "Warning: LTTNG_UST_ZE_PARANOID_DRIFT not activated without LTTNG_UST_ZE_PROFILE\n");
   }
+
+  if (getenv("LTTNG_UST_SAMPLING_ENERGY")) {
+    zerInit();
+    interval.tv_sec = 0;
+    interval.tv_nsec = 50000000;
```

Comment:
Maybe just read the value of `LTTNG_UST_SAMPLING_ENERGY` or add a new parameter to overwrite this.